### PR TITLE
What's new in v6.2

### LINKED ISSUE #16902: Docs: What's new article for v6.2
What's new article for v6.2

### PR DESCRIPTION
**Pull request by marefr**
Very rough outline/draft of What's new in v6.2.

Closes #16902

### CODE REVIEW

File: docs/sources/guides/whats-new-in-v6-2.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+### Official support for Elasticsearch 7
+
+Grafana v6.2 ships with official support for Elasticsearch v7, see [Using Elasticsearch in Grafana](/features/datasources/elasticsearch/#elasticsearch-version) for more information.
+
+### Minor Features and Fixes
+
+This release contains a lot of small features and fixes:
+
+- Panels: No title will no longer make panel header take up space
```
Comment (dprokop) [inline docs/sources/guides/whats-new-in-v6-2.md]: `Empty title` ?

File: docs/sources/guides/whats-new-in-v6-2.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+- Explore - Adds user time zone support, reconnect for failing datasources and a fix that prevents killing Prometheus instances when Histogram metrics are loaded.
+- Alerting - Adds support for configuring timeout durations and retries, see [configuration](/installation/configuration/#evaluation-timeout-seconds) for more information.
+- Elasticsearch - A small bug fix to properly display percentiles metrics in table panel.
+- InfluxDB - Support for POST HTTP verb.
+- CloudWatch - Important fix for default alias disappearing in v6.1.
```
Comment (dprokop) [inline docs/sources/guides/whats-new-in-v6-2.md]: `- Search - Works in a scope of a viewed dashboard's folder`

Comment (marefr) [reply docs/sources/guides/whats-new-in-v6-2.md]: @dprokop didn't see that PR with a "add to changelog" label - can you add that to PR?

File: docs/sources/guides/whats-new-in-v6-2.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+weight = -13
++++
+
+# What's New in Grafana v6.2
+
+## Highlights
+
+### Improved security
+
+- Ensure encryption of datasource secrets
```
Comment (aocenas) [inline docs/sources/guides/whats-new-in-v6-2.md]:
```
Datasource can and should use secureJsonData to store password and basicAuthPassword fields. For existing datasources this is an opt in feature that needs a manual step. See [docs](/installation/upgrading/#upgrading-to-v6-2) for migration details.
```

Comment (aocenas) [reply docs/sources/guides/whats-new-in-v6-2.md]: Ah I see @bergquist already did some updates with links.

File: docs/sources/guides/whats-new-in-v6-2.md (hunk 2 above)
Comment (dprokop) [reply docs/sources/guides/whats-new-in-v6-2.md]: yas

**Review by bergquist — APPROVED** [review]
LGTM

**Pull request merged**